Formula/konoha.rb

```diff
@@ -3,6 +3,7 @@ class Konoha < Formula
   homepage "https://github.com/konoha-project/konoha3"
   url "https://github.com/konoha-project/konoha3/archive/v0.1.tar.gz"
   sha256 "e7d222808029515fe229b0ce1c4e84d0a35b59fce8603124a8df1aeba06114d3"
+  revision 1
```

Comment:
Yeah, that's what I had in mind.

For whatever reason, `konoha` links both `libmpi.12.dylib` and `libmpi_cxx.1.dylib` from:
```
lib/minikonoha/0.1/package/konoha.mpi/mpi_glue.dylib
```
So the formula needed a bump to move over to the updated links.